fpm/src/fpm_compiler.f90

```diff
@@ -53,8 +56,8 @@ subroutine add_compile_flag_defaults(build_name,compiler,model)
        & -ffast-math&
        & -funroll-loops&
        &'
+       mandatory=' -J '//modpath//' -I '//modpath ! add module path as apprpriate
```

Comment:
Here is a bulk copy-and-paste typo in apprpriate -> appropriate. While there 25 copies of this typo below, I would recommend to remove the comment altogether and add the explanation to the top of the select case instead.